docker/api/Dockerfile

```
@@ -12,27 +12,30 @@ LABEL org.opencontainers.image.vendor="Cybersecurity and Infrastructure Security
 
 ARG CISA_UID=421
 ENV CISA_HOME="/home/cisa"
-ENV ECHO_MESSAGE="Hello World from Dockerfile"
 
 RUN addgroup --system --gid ${CISA_UID} cisa \
   && adduser --system --uid ${CISA_UID} --ingroup cisa cisa
 
-RUN apk --update --no-cache add \
-ca-certificates \
-openssl \
-py-pip
+# RUN apk --update --no-cache add \
+
+RUN apt-get update && \
+ apt-get install wget
+
+# ca-certificates \
+# openssl \
+# py-pip
 
 WORKDIR ${CISA_HOME}
 
-RUN wget -O sourcecode.tgz https://github.com/cisagov/skeleton-python-library/archive/v${VERSION}.tar.gz && \
+RUN wget -O sourcecode.tgz https://github.com/cisagov/li-pca-app/archive/refs/tags/v0.0.1-rc2.tar.gz && \
   tar xzf sourcecode.tgz --strip-components=1 && \
-  pip install --requirement requirements.txt && \
-  ln -snf /run/secrets/quote.txt src/example/data/secret.txt && \
+  pip3 install -e . && \
```

Is there a reason you aren't using the `requirements.txt` file? You at least need to use the long syntax:

```suggestion
  pip3 install --editable . && \
```

---

👍 I believe we found that in a repo we were using for reference and just kept the formatting, but I agree that we should at least use the long syntax for clarity. We will likely update this as the project grows and we add more things in requirements.